datacapture/src/main/java/com/google/android/fhir/datacapture/views/GroupHeaderView.kt

```diff
@@ -37,19 +36,22 @@ internal class GroupHeaderView(context: Context, attrs: AttributeSet?) :
     LayoutInflater.from(context).inflate(R.layout.group_type_header_view, this, true)
   }
 
-  fun bind(questionnaireItem: Questionnaire.QuestionnaireItemComponent) {
-    val prefix = findViewById<TextView>(R.id.prefix)
-    val question = findViewById<TextView>(R.id.question)
-    val hint = findViewById<TextView>(R.id.hint)
+  private val prefix = findViewById<TextView>(R.id.prefix)
+  private val question = findViewById<TextView>(R.id.question)
+  private val hint = findViewById<TextView>(R.id.hint)
+
+  fun bind(questionnaireViewItem: QuestionnaireViewItem) {
     initHelpViews(
       helpButton = findViewById(R.id.helpButton),
       helpCardView = findViewById(R.id.helpCardView),
       helpTextView = findViewById(R.id.helpText),
-      questionnaireItem
+      questionnaireViewItem.questionnaireItem
```

Comment:
if you are listing the parameter names of the variables of the function, list them all:
```suggestion
      questionnaireItem = questionnaireViewItem.questionnaireItem
```

Reply:
Done. Added `displayItems_shouldGetEnabled_withAnswerChoice` in `QuestionnaireUiEspressoTest`.